prov/net/src/xnet_progress.c

```diff
@@ -46,6 +46,21 @@
 static ssize_t (*xnet_start_op[ofi_op_write + 1])(struct xnet_ep *ep);
 
 
+static inline void xnet_active_ep(struct xnet_ep *ep)
+{
+	ep->progress_cnt++;
+	if (ep->is_active || ofi_poll_fairness)
```

Comment:
I believe this should be `if (ep->is_active || ofi_poll_fairness)`, otherwise if ep is not active and ofi_poll_fairness is set, we will still never activate the ep.

Reply:
I *think* you meant ```if (ep->is_active || !ofi_poll_fairness)``` ?

Reply:
Oops, yes. I did.